User-Help/AskingGuidance.md

```diff
@@ -0,0 +1,37 @@
+## How to ask a great question
+
+The best way to get a perfect answer to your questions is to ask a great question. The better your question is, the easier it will be for someone to be able to give you the answer you need. Here are a few things to keep in mind when asking questions.
+
+**Make sure your question is in scope**
+
+Every Codidact site has its own guidelines for what topics are able to be asked about on that site. You wouldn't want to ask about frying eggs on [Writing](https://writing.codidact.com), or bicycle maintainence on [Cooking](https://cooking.codidact.com). See [$SiteName's scope guidelines here](/help/scope).
+
+If your question isn't in scope for this site, you can check out our [list of communities](https://codidact.com) to see if another site fits your needs. If what you're looking for isn't there, you can always propose a new site on [Codidact Meta](https://meta.codidact.com).
+
+**Do some research**
+
+Before asking a new question, first take a look around. Has your question been asked before here on Codidact? You can [do a search on Codidact](/help/search) for keywords related to your question, or take to your favorite search engine to see if your answer is already there on Codidact.
+
+In addition to just making sure your question hasn't been asked already here on Codidact, take a few moments to search beyond the site. If you put your question title into a search engine, can you find the answer to your question in the first three results? If so, perhaps consider [alternative ways of sharing that information](/help/articles-blogs) here on Codidact, or writing a self-answered question to share that knowlege.
+
+**Include your end goal**
+
+It's always a good idea to include what exactly you're trying to achieve when you ask a question. It helps to provide context to your question, and can influence answers in ways you possibly hadn't considered.
+
+**Avoid too much subjectivity**
+
+Questions like "What's your favorite way to eat an egg?" can't be answered objectively - the answer, naturally, changes depending on who you ask. On the other hand, "What's the difference between hard-boiled eggs and an omelette?" can be answered without relying too much on personal opinions. Try to ask questions that can be answered with some level of objectivity.
+
+**Be specific**
+
+Asking really broad questions often isn't very useful. Instead, narrow your question down to ask about one thing in particular. Asking "How do I care for a dragon?" would need several full-size books to answer, but "Is garlic safe for my dragon to eat?" is much more manageable. Be specific when asking a question, and don't ask questions that would need answers the size of books.
```

Comment:
I suggest continuing the "from our world" example in the previous section instead of switching over to dragons.  Cooking example: "how do I care for a basil plant" is broad; "how much should I harvest at a time and where should I take it from" is specific.